### PR TITLE
Taras yarmus

### PR DESCRIPTION
**Pull request by kolyasalubov**

### CODE REVIEW

File: HW5/TarasYarmus/12.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+question = input(
+    '''
+    Hello! I will calculate square by your arguments! 
+    Calculate square for rectangle: press 1.
+    Calculate square for triangle: press 2.
+    Calculate square for circle: press 3.
+    Make your choice: ''')
+
+if question == '1':
+    rectangle1 = float(input('Enter first side length: '))
```
Comment (kolyasalubov) [inline HW5/TarasYarmus/12.py]: it is better to move these lines to the appropriate functions

File: HW5/TarasYarmus/12.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    rectangle2 = float(input('Enter second side length: '))
```
Comment (kolyasalubov) [inline HW5/TarasYarmus/12.py]: it is better to move these lines to the appropriate functions

File: HW5/TarasYarmus/12.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    result_rectangle = rectangle(rectangle1, rectangle2)
+    print(f'Square of your rectangle is: {result_rectangle}')
+elif question == '2':
+    triangle1 = float(input('Enter your height: '))
```
Comment (kolyasalubov) [inline HW5/TarasYarmus/12.py]: it is better to move these lines to the appropriate functions

File: HW5/TarasYarmus/12.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    triangle2 = float(input('Enter your basis: '))
```
Comment (kolyasalubov) [inline HW5/TarasYarmus/12.py]: it is better to move these lines to the appropriate functions

File: HW5/TarasYarmus/12.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    result_triangle = triangle(triangle1, triangle2)
+    print(f'Square of your triangle is: {result_triangle}')
+elif question == '3':
+    circle1 = float(input('Enter numeric for "r": '))
```
Comment (kolyasalubov) [inline HW5/TarasYarmus/12.py:47]: it is better to move these lines to the appropriate functions

File: HW5/TarasYarmus/12.py (hunk 1 above)
Comment (Kudik01) [reply HW5/TarasYarmus/12.py]: Moved to the appropriate function.

File: HW5/TarasYarmus/12.py (hunk 2 above)
Comment (Kudik01) [reply HW5/TarasYarmus/12.py]: Moved to the appropriate function.

File: HW5/TarasYarmus/12.py (hunk 3 above)
Comment (Kudik01) [reply HW5/TarasYarmus/12.py]: Moved to the appropriate function.

File: HW5/TarasYarmus/12.py (hunk 4 above)
Comment (Kudik01) [reply HW5/TarasYarmus/12.py]: Moved to the appropriate function.

File: HW5/TarasYarmus/12.py (hunk 5 above)
Comment (Kudik01) [reply HW5/TarasYarmus/12.py:47]: Moved to the appropriate function.

**Pull request merged**